Sources/Basics/FileSystem/NativePathExtensions.swift

```diff
@@ -23,6 +28,24 @@ extension AbsolutePath {
             return repr
         }
     }
+
+    /// Returns the `pathString` on non-Windows platforms.  On Windows
+    /// platforms, this provides the path string normalized as per the Windows
+    /// path normalization rules.  In the case that the path is a long path, the
+    /// path will use the extended path syntax (UNC style, NT Path).
+    public var _normalized: String {
```

Comment:
should this be the default behavior of `pathString` on windows?

Reply:
I've been thinking about it, but I want to fix up the test suite first and then test things before making such a change.  That will allow us to ensure that we don't accidentally regress something.  In the mean time, this would repair the use of SPM.

Reply:
in that case I think you should move `_normalized` closer to the single call site, and if we have a second one we make it the default behavior. wdyt?

Reply:
Sure, so you want to move it into that file?  I'm totally fine with that.